### PR TITLE
refactor: remove unnecessary or operation

### PR DESCRIPTION
**Pull request by anonrig**
Unless I'm not seeing something obvious, this OR statement shouldn't be required.

### CODE REVIEW

**Comment by anonrig** [comment]
It seems like the automatic git hook fixes a listing error while I commit...

**Review by roae — APPROVED** [review]
Nice catch, thank you @anonrig! :pray:

File: packages/trpc/server/routers/viewer/bookings.tsx [hunk 1]
```diff
@@ -145,15 +145,7 @@ export const bookingsRouter = router({
         },
         unconfirmed: {
           endTime: { gte: new Date() },
-          OR: [
-            {
-              recurringEventId: { not: null },
```
Comment (zomars) [inline packages/trpc/server/routers/viewer/bookings.tsx:150]: Recurring events are displayed in another tab, so this condition is needed.

**Pull request merged**